openapi/endpoints/simulator/models/fxReversalAmount.yaml

```diff
@@ -0,0 +1,12 @@
+type: object
+description: fxAmount can be specified to process the transaction in a currency other than USD
+required:
+  - amount
+  - currencyCode
+properties:
+  amount:
+    type: string
+    description: Amount to reverse on the prior authorize or clear requests specified in the acquirers currency. An integer in the smallest denomination for the given currency. This can be generated by converting the `reversalAmount`
```

Comment:
✏️ "converting the reversalAmount" reads a bit odd. Like the sentence is incomplete. Could we maybe elaborate a little what kind of conversion is expected?